pixart.js

```diff
@@ -1 +1,16 @@
+var setColors =document.querySelector(".color");
+var colorField =document.querySelector(".square");
+var brush = document.querySelector(".brush");
 
+$("button")[0].addEventlistener("clicks", handleClickEvent);
+$("colorField")[0].addEventlistener("clicks", handleClickEvent);
+
+for (var i = 0; i < 20; i ++){
+  var = div.createElement("div");
```

Comment:
Please review the docs for createElement.  Once you create them, you still have to associate them with an element that is on the page.